docs/labs/flockercontainers.rst

```diff
@@ -0,0 +1,96 @@
+.. _flocker-containers:
+
+=============================
+Running Flocker in Containers
+=============================
+
+The :ref:`Flocker Installer <labs-installer>` allows you install Flocker in a variety of environments.
+For some of these environments, such as CoreOS, you can actually install Flocker inside a container.
```

Comment:
:question: I suggest "you *must*  install Flocker *services* inside a containers."